demos/gaze_estimation_demo/cpp_gapi/main.cpp

```diff
@@ -290,33 +287,28 @@ int main(int argc, char *argv[]) {
                 inferenceResults.push_back(inferenceResult);
             }
 
-            /** Measure FPS **/
-            auto tIterationEnds = cv::getTickCount();
-            double overallTime = (tIterationEnds - tIterationBegins) * 1000. / cv::getTickFrequency();
-            overallTimeAverager.updateValue(overallTime);
-            tIterationBegins = tIterationEnds;
-
-            /** Print logs **/
-            if (FLAGS_r) {
-                for (auto& inferenceResult : inferenceResults) {
-                    slog::debug << inferenceResult << slog::endl;
-                }
+            /** FlipImage **/
```

Comment:
I wondering why? Put this `if()` after /** Display the results **/ `for () ...`

Reply:
Reverted in https://github.com/openvinotoolkit/open_model_zoo/pull/2647/commits/e4847b2a2954467da620c9d710d496c3686eeb26